.github/workflows/Tests.yml

```diff
@@ -48,36 +50,33 @@ jobs:
     name: ${{ matrix.type }} | ${{ matrix.test }}
 
     env:
-      PDK_ROOT: $GITHUB_WORKSPACE/rules/klayout
-      PDK: ${{ matrix.pdk }}
+      IMAGE: ghcr.io/google/globalfoundries-pdk-libs-gf180mcu_fd_pr/test
```

Comment:
is the image public?

Reply:
The image is neither public nor private, because it's not pushed to any registry. It's built in one step and used in the next step, both of them executed within the same job. Therefore, the name might be anything; it's local only.

However, it might make sense to eventually split building the image and using it, in order to avoid installing the dependencies in each job. That's what #55 is about.

Reply:
ah make sense, curious if you consider caching it with `docker save/load` instead?